csm/core/controllers/cluster_management.py

```diff
@@ -108,3 +108,21 @@ async def get(self):
                                         .get_cluster_status(node_id)
         return cluster_status
 
+@CsmView._app_routes.view("/api/v2/system/management/{resource}/{id}")
+class ClusterResourceStatusView(CsmView):
+    def __init__(self, request):
+        super().__init__(request)
+        self.cluster_management_service = self.request.app[const.CLUSTER_MANAGEMENT_SERVICE]
+
+    @CsmAuth.permissions({Resource.CLUSTER_MANAGEMENT: {Action.LIST}})
+    @Log.trace_method(Log.DEBUG)
+    async def get(self):
+        """
+        API to get cluster resource status with id
```

Comment:
API to get cluster resource status with id --> This will get the health of a resource in the system